rootfs/extra/etc/nginx/njs/http/cors.js

```
@@ -0,0 +1,146 @@
+		break;
+	    }
+	}
+    }
+    if ('' === access_control_allow_origin) {
+	return; // there were no matching allowed origins - stop CORS processing
+    }
+
+    // Access-Control-Allow-Origin
+    r.headersOut['Access-Control-Allow-Origin'] = access_control_allow_origin;
+    if ('Vary' in r.headersOut) {
+	let elements = r.headersOut['Vary'].split(', ');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'elements' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)